types/audit-details.d.ts

```diff
@@ -161,6 +161,7 @@ declare global {
         path?: string;
         selector?: string;
         snippet?: string;
+        nodeLabel?: string;
```

Comment:
also add a comment here?

`/** A text descriptor for the node intended to help quickly recognize which node is being referenced. */`

or something